src/webui/www/public/scripts/dynamicTable.js

```diff
@@ -488,18 +488,16 @@ var DynamicTable = new Class({
             this.selectedRows.empty();
         },
 
-        selectRow : function (rowId) {
-            this.selectedRows.empty();
+        selectRow : function (rowId, clear) {
+            if (clear !== false)
+                this.selectedRows.empty();
             this.selectedRows.push(rowId);
             var trs = this.tableBody.getElements('tr');
             for (var i = 0; i < trs.length; i++) {
```

Comment:
Although not your code, I would change it to `array.each()`.

Reply:
That's a dangerous rabbit hole to go down because there are many instances of `for` loops that can be replaced with `.each()` just in this file. I'll modify this function now, but we should try to target these in a future PR.

Reply:
ok, thanks.